spec/requests/shared/stats_spec.rb

```
@@ -3,10 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe 'Shared::Stats', type: :request do
-  before do
-    stub_request(:any, 'https://api.github.com/repos/Freika/dawarich/tags')
-      .to_return(status: 200, body: '[{"name": "1.0.0"}]', headers: {})
-  end
+
 
```

RuboCop (1.81.7)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)